ytmusicapi/ytmusic.py

```diff
@@ -100,12 +100,17 @@ def __init__(self,
                     "The provided credentials are invalid. Reason given by the server: "
                     + response['error']['status'])
 
-    def _send_request(self, endpoint: str, body: Dict, additionalParams: str = "") -> Dict:
+    def _send_request(self,
```

Comment:
There should be no need to modify this file as part of this PR. I recommend to use the battle-tested get_continuations function here: https://github.com/sigma67/ytmusicapi/blob/ca8a73046bb43441d8e40a84569e0383bcb305f6/ytmusicapi/parsers/utils.py#L50-L59

For usage, refer for example to https://github.com/sigma67/ytmusicapi/blob/5a1032983d1a665019944957aeb5eb4886e799b0/ytmusicapi/mixins/library.py#L8

It has all the continuation logic you need. This avoids duplicate code in the project, which currently there is quite a bit of in this PR. I have marked it where applicable.

Reply:
Great - I'll give this a go.  Thanks.